apps/web/.env.production

```
@@ -7,6 +7,8 @@
 # Override with Vercel environment variables if needed.
 BACKEND_URL=https://uvai-backend-gpwz4wb5na-uc.a.run.app
 NEXT_PUBLIC_BACKEND_URL=https://uvai-backend-gpwz4wb5na-uc.a.run.app
+UVAI_API_RATE_LIMIT_PER_MINUTE=60
+UVAI_AI_RATE_LIMIT_PER_MINUTE=12
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The UVAI_AI_RATE_LIMIT_PER_MINUTE key should go before the UVAI_API_RATE_LIMIT_PER_MINUTE key

(UnorderedKey)